src/CADL.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -263,6 +266,7 @@ export function getInputType(
             // In such cases, we don't want to emit a ref and instead just
             // emit the base type directly.
             default:
+                const sdkType = getSdkSimpleType(context, type);
```

Comment:
sdkType isn't used anywhere from what i can tell, i think we should pass it in to `getCSharpInputTypeKindByIntrinsicModelName(sdkType.kind)`. Might have to change some of the code in the mapping between the two